vis_sink.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018 Universidad de Costa Rica, Autonomous Robots
+# and Cognitive Systems Laboratory (ARCOS-lab)
+# Author: Daniel Garcia-Vaglio <degv364@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""This creates a scene logger.
+
+Opens the same ports required for visualization
+but it logs the info instead of drawing it.
+"""
+from __future__ import print_function
+
+from builtins import zip
+from builtins import str
+from builtins import map
+from builtins import range
+import time
+import sys
+import signal
+import yarp
+import optparse
+import pickle
+from numpy import array
+
+from arcospyu.control.control_loop import Controlloop
+
+sys.path.append('../hand_cart')
```

Comment:
This depends on the file being ran form a specific directory or else this will fail...

I don't see a good instance where this is a good idea. I know a lot of our repos has things like this one but lets not copy bad methods.

Maybe passing the path to this as a commandline option?